test/test_mkldnn.py

```diff
@@ -216,6 +216,55 @@ def test_add(self):
         torch.add(mx, my, alpha=alpha, out=mkldnn_out)
         self.assertEqual(out, mkldnn_out.to_dense())
 
+    def test_mul(self):
+        N = torch.randint(3, 10, (1,)).item()
```

Comment:
Please do not relay on random numbers.  You are not freezing random seed, that means every test execution will generate new shape. It might create false sense of security, but many modern test frameworks will attempt to rerun failed test hiding problem prom developers.

Much better if you explicitly specify shapes you are interested to test.